doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+**Input:**
+
+::
+  
+  array_example.size
+  
+
+**Output:**
+
+::
+
+  24
+
+And to find the shape of your array, run:
+
+**Input:**
+
+::
+
+  array_example.shape
+
+**Output:**
+
+::
+
+  (3,2,4)
+
+`Read more about dimensions here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.ndim.html>`_, `size here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.size.html>`_, and `shape here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.shape.html>`_.
+
+Can you reshape an array?
+-------------------------
+
+
+::
+
+  np.reshape()
+
+-----
+  
+**Yes!**
+
+Using ``np.reshape()`` will give a new shape to an array without changing the data. Just remember that when you use the reshape method, the array you want to produce needs to have the same number of elements as the original array. If you start with an array with 12 elements, you'll need to make sure that your new array also has a total of 12 elements.
+
+If you start with this array:
+
+::
+
+  a = np.arange(6)
+  print(a)
+
+**Output:**
+
+::
+
+  [0 1 2 3 4 5]
+
+You can use ``reshape()`` to reshape your array. For example, you can reshape this array to an array with three rows and two columns:
+
+**Input:**
+
+::
+
+  b = a.reshape(3,2)
+  print(b)
+
+**Output:**
+
+::
+
+  [[0 1]
+   [2 3]
+   [4 5]]
+
+With ``np.reshape``, you can specify a few optional parameters:
+
+**Input:**
+
+::
+
+  numpy.reshape(a, newshape, order)
+
+``a`` is the array to be reshaped.
+
+``newshape`` is the new shape you want. You can specify an integer or a tuple of integers. If you specify an integer, the result will be an array of that length. The shape should be compatible with the original shape.
+
+``order:`` ``C`` means to read/write the elements using C-like index order,  ``F`` means to read/write the elements using Fortran-like index order, ``A`` means to read/write the elements in Fortran-like index order if a is Fortran contiguous in memory, C-like order otherwise. (This is an optional parameter and doesn't need to be specified.)
+
+`Learn more about shape manipulation here <https://docs.scipy.org/doc/numpy-1.17.0/user/quickstart.html#shape-manipulation>`_.
+
+
+How to convert a 1D array into a 2D array (how to add a new axis to an array)
+-----------------------------------------------------------------------------
+
+::
+
+  np.newaxis
+  np.expand_dims
+
+-----
+
+You can use ``np.newaxis`` and ``np.expand_dims`` to increase the dimensions of your existing array.
+
+Using ``np.newaxis`` will increase the dimensions of your array by one dimension when used once. This means that a **1D** array will become a **2D** array, a **2D** array will become a **3D** array, and so on. 
+
+For example, if you start with this array:
+
+::
+
+  a = np.array([1, 2, 3, 4, 5, 6])
+  a.shape
+
+**Output:**
+
+::
+
+  (6,)
+
+You can use ``np.newaxis`` to add a new axis:
+
+**Input:**
+
+::
+
+  a2 = a[np.newaxis]
```

Comment:
I'd always write it as `a2 = a[np.newaxis, :]` for clarity

Reply:
Ah, that's done below. Perhaps duplicate?

Reply:
Got it!